src/features/search/components/EmailObfuscation.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+import { Button, Box, Link } from "@chakra-ui/react";
+
+const EmailButton: React.FC<{ email: string }> = ({ email }) => {
```

Comment:
żeby być spójnym z resztą kodu to wolałbym otypować te propsy po prostu jako {email} : {email: string}

i zamiast export default na dole zrób tutaj po prostu

```tsx
export const EmailButton ...
```